### DISCUSSION TITLE
Dose Normalization from DOSXYZnrc ouput by using isource 3

### DISCUSSION THREAD

**Discussion by AnnisaSW** [nrc-cnrc/EGSnrc#1091] [Questions]
so i have running the simulation by using DOSXYZnrc by using isouce 3 and i got the result from that in 3ddose file. from that result, i want to normalize the dose. from the user manual i should normalized by the incident particle fluence. like this : 
![image](https://github.com/nrc-cnrc/EGSnrc/assets/156294740/994ff3e1-0759-41c3-8b01-575479e3c442)
but i dont know how to get the nmissm value. so can anyone tell me how to get that?

**Comment by rtownson** [comment]
This is just explaining the normalization that is already done for you, by the code. The results in the 3ddose file are already normalized per incident particle fluence. Generally it is not necessary to convert directly to `Gy`, but there are papers out there on absolute dose normalization.
